api/net/Areas/Editor/Controllers/LookupController.cs

```diff
@@ -136,7 +136,7 @@ public async Task<IActionResult> FindAllAsync()
         var metrics = _metricService.FindAll();
         var tagServices = _tagService.FindAll();
         var tonePools = _tonePoolService.FindAll();
-        var users = _userService.FindAll();
+        var users = _userService.FindByRoles(roles.Where(x => x.ToLower() != ClientRole.Subscriber.ToString().ToLower()));
```

Comment:
This is incorrect.  We want to return all users who are part of the Editor role.

Reply:
No need to `ToLower()` as you are doing it in the service function

Reply:
Updated, and here we are retrieving the roles other than "subscriber", which means for both "editor" and "administrator" in this case. Thanks